src/components/Icons/SearchIcon.jsx

```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types';
 const SearchIcon = ({
   fill, width, height, viewBox,
 }) => (
-  <svg width={width} height={height} viewBox={viewBox} fill={fill} xmlns="http://www.w3.org/2000/svg">
+  <svg style={{ display: 'block' }} width={width} height={height} viewBox={viewBox} fill={fill} xmlns="http://www.w3.org/2000/svg">
```

Comment:
No, you shouldn't write such styles here. You can apply this styles in `IconContainer` styled-component like this:
```js
export const IconContainer = styled...`
  position: relative;
  ...

  svg {
    display: block;
  }
`
```